bin/git-delete-merged-branches

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-branches=$(git branch --no-color --merged | grep -v "\*" | grep -v master | grep -v svn)
+branches=$(git branch --no-color --merged | grep -v "\*" | grep -vE "\b(master|dev|develop)\b" | grep -v svn)
```

Comment:
protect `develop` branch as well as `master` and match only on whole-words so that branches like `feature/IAmTheGateKeeperAreYouTheKeymaster` aren't inadvertently spared from being cleaned up.